libpod/container_internal_linux.go

```diff
@@ -2445,6 +2445,12 @@ func (c *Container) getOCICgroupPath() (string, error) {
 		// slice:prefix:name
 		systemdCgroups := fmt.Sprintf("%s:libpod:%s", path.Base(c.config.CgroupParent), c.ID())
 		logrus.Debugf("Setting CGroups for container %s to %s", c.ID(), systemdCgroups)
+		// If user is rootless and no valid systemd user session is present then break early.
+		if rootless.IsRootless() && unified {
+			if !utils.CheckifSystemdSessionisValid(rootless.GetRootlessUID()) {
+				logrus.Infof("Invalid systemd user session")
+			}
+		}
```

Comment:
is this something we can do just when creating the user namespace (i.e. just at startup for each user)?  I am afraid we are adding an additional cost at every Podman invocation just to provide an info message.

Reply:
@giuseppe I agree, initially PR was to fail if no valid session is found but later it was changed to warning and this much facade and cost is too much for a warning I'll do the relevant changes.

Reply:
Made changes in latest commit.